docs/framework/react/guides/ssr.md

```diff
@@ -25,7 +25,7 @@ This section focuses on integrating TanStack Form with TanStack Start.
 Let's start by creating a `formOption` that we'll use to share the form's shape across the client and server.
 
 ```typescript
-// app/routes/index.tsx, but can be extracted to any other path
+// src/utils/form-isomorphic.ts, but can be extracted to any other path
```

Comment:
What could be a good comment for specifying that we shouldn't put it on any routes file?